nl.json

```diff
@@ -72,19 +72,19 @@
             "home" : "home",
             "shop" : "winkel",
             "cosmetics" : {
-                "dropdown" : "cosmetica",
-                "list" : "list &amp; zoeken",
+                "dropdown" : "cosmetisch",
```

Comment:
Not entirely sure why you changed this, but the English translation uses an plural noun, and this is an singular noun.